src/main/java/bookstore/dto/CreateBookRequestDto.java

```diff
@@ -0,0 +1,16 @@
+package bookstore.dto;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
Is this annotation required?

Reply:
No i've seen that in Bohdan`s video and just added it to avoid adding id to book entity from postman